helm/ffc-pay-processing/values.yaml

```diff
@@ -49,6 +49,11 @@ container:
   debtTopicAddress: ffc-pay-debt-data
   qcTopicAddress: ffc-pay-quality-check
```

Comment:
These names are tidied up in a subticket elsewhere but for this PR to work, `qcTopicAddress` needs to be `ffc-pay-debt-data-response`